some basic docker usage

reusr1 opened the pull request:
some hints about how to use docker

reusr1 committed:
some basic docker usage

GastonGonzalez reviewed (changes requested):
Please add Docker remove commands to allow testing new branches.

reusr1 committed:
updated docs due to comments

reusr1 commented:
> Please add Docker remove commands to allow testing new branches.

@GastonGonzalez I added the docker remove with -rm to the run command. I added a bit of explanation to each command to ease people without a lot of docker knowledge into it a bit more.

ehdatheadwire commented on docs/public/docker/index.md:
````diff
@@ -0,0 +1,39 @@
+replace the {number} in the command below with the PR number for the change you would like to inspect
+
+```bash
+docker run -it -p 8080:8080 peregrinecms/peregrinecms:pr-{number}
````
Don't we also have to replace 8080 with a made-up number that is greater than 9000? e.g. 9209 for pr-209?

reusr1 committed:
explain changing ports a bit more

ehdatheadwire commented:
replace all instances of /peregrinecms with /peregrine-cms -- through testing the new instructions I discovered that this missing dash is what was causing my error messages while testing a pr yesterday :)

ehdatheadwire reviewed (approved):
Recommended adjustments made. Looks good from this view!
